chapters/connectors.tex

```diff
@@ -130,26 +128,25 @@ \subsection{Expandable Connectors}\doublelabel{expandable-connectors}
   component, and if the other connector is an undeclared element in a
   declared expandable connector it is handled as follows (elements that
   are only potentially present are not seen as declared):
-\begin{itemize}
-\item
-  The expandable connector instance is automatically augmented with a
-  new component having the used name and corresponding type.
-\item
-  If the undeclared component is subscripted, an array variable is
-  created, and a connection to the specific array element is performed.
-  Introducing elements in an array gives an array with at least the
-  specified elements, other elements are either not created or have a
-  default value (i.e.\ as if they were only potentially present).
-\item
-  If the variable on the other side of the connect-equation is input or
-  output the new component will be either input or output to satisfy the
-  restrictions in \autoref{restrictions-of-connections-and-connectors} for a non-expandable connector.
-  \begin{nonnormative}
-  If the existing side refers to an inside connector (i.e.\ a connector of a component) the new variable will copy its causality, i.e.\ input
-  if input and output if output, since the expandable connector must be an outside connector.
-  \end{nonnormative}
-  For an array the input/output property can be deduced separately for each array element.
-\end{itemize}
+  \begin{itemize}
```

Comment:
i don't understand why these lines are indented one space; and that also makes it hard to see the diff.

Reply:
The problem is that the source gets very confusing to real when nested item lists are using the same indentation.